fix_vector_store_rag.py

```
@@ -0,0 +1,45 @@
+"""Script to fix the Vector Store RAG.json file."""
+
+import json
+from pathlib import Path
+
+file_path = "src/backend/base/langflow/initial_setup/starter_projects/Vector Store RAG.json"
+
+
+def fix_json_file(file_path):
+    """Fixes the Vector Store RAG.json file by removing the parser node and specific AstraDB fields."""
+    with Path(file_path).open("r+") as f:
```

Pylint (3.3.7)

[warning] Using open without explicitly specifying an encoding

(W1514)